plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/factories/OrderedEContentProvider.java

```diff
@@ -83,6 +98,16 @@
 	}
 
 	static private class InternalFactoryDispatcherX extends N4JSXSwitch<List<EObject>> {
-		// stub for future order adjustments in N4JSX elements
+
+		// TODO: Remove this and change order in N4JSX.ecore
```

Comment:
why TODO, why change order?

Reply:
This will be done, after N4JSX is merged. The order of the xcore-file is important for the flowgraphs. If the xcore-file defines a wrong order (w.r.t. the order of execution), this wrong order is overwritten here.